patches/third_party-blink-renderer-modules-canvas-canvas2d-base_rendering_context_2d.cc.patch

```diff
@@ -91,7 +91,9 @@ index 783f5c3e3ddf5c33f43b9cccc4070cfc7c866e86..fc24e3a739e6e73b734c6506888cc620
      int sh,
      ExceptionState& exception_state) {
 +  LocalDOMWindow* window = LocalDOMWindow::From(script_state);
-+  if (window && !AllowFingerprinting(window->GetFrame())) return nullptr;
++  if (window && !AllowFingerprinting(window->GetFrame())) {
++    return ImageData::CreateForTest(IntSize(0, 0));
```

Comment:
I'm using this because this one would allow size 0 while other create functions don't, and could patch less code compared to normal create functions.